lib/widgets/matrix.dart

```diff
@@ -351,13 +353,14 @@ class MatrixState extends State<Matrix> with WidgetsBindingObserver {
           ),
         );
 
-        if (state != LoginState.loggedIn) {
-          navigatorContext?.go('/rooms');
-        }
+        navigatorContext?.go('/rooms');
       } else {
         setUpToMServicesInLogin(c);
         if (state == LoginState.loggedIn) {
           navigatorContext?.go('/rooms');
+        } else {
+          await getIt.get<HiveCollectionToMDatabase>().clear();
+          await c.clear();
```

Comment:
maybe we dont need it? WDYT?

Reply:
For me we don't need it because it's already call in logout action

Reply:
please remove it